### ISSUE TITLE
Page issue: Huion HS610

### ISSUE THREAD

**Issue by AstolfoKawaii** [DIGImend/tablets#32]
There is an issue with page [Huion HS610](http://digimend.github.io/tablets/Huion_HS610/):

As I understood from https://github.com/DIGImend/digimend-kernel-drivers/issues/275 HS610 is supported.

**Comment by diplomatichal** [comment]
Indeed, it would seem so. @spbnick do you confirm? Thanks a lot for the great support!
